src/test/java/Homework18.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+
+
+public class Homework18 extends BaseTest{
+
+    @Test
+ public void PlayaSong(){
+
+        provideEmail("demo@class.com");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        playsong();
+        isSongPlaying();
+
+ }
+public void playsong(){
+    WebElement PlayNextButton = driver.findElement(By.xpath("//*[@title='Play next song']"));
+    WebElement PlayButton = driver.findElement(By.xpath("//span[@data-testid='play-btn']"));
+    PlayNextButton.click();
+    PlayButton.click();
+}
+public void isSongPlaying(){
+WebElement SoundBar = driver.findElement(By.xpath("//div[@data-testid='sound-bar-play']"));
+    Assert.assertTrue(SoundBar.isDisplayed());
+
+}
+}
```

Comment:
**How can I beautify code in IntelliJ:**

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)